tasks/sep/020-holocene-system-config-upgrade-multi-chain/OVERVIEW.md

```diff
@@ -0,0 +1,85 @@
+# Holocene Hardfork - SystemConfig Upgrade
+Upgrades the `SystemConfig.sol` contract for Holocene.
+
+The batch will be executed on chain ID `11155111`, and contains `5` transactions.
+
+## Tx #1: Upgrade `SystemConfig` proxy for OP Sepolia
+Upgrades the `SystemConfig` proxy to the new implementation, featuring configurable EIP-1559 parameters.
+
+**Function Signature:** `upgrade(address,address)`
+
+**To:** `0x189aBAAaa82DfC015A588A7dbaD6F13b1D3485Bc`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x99a88ec4000000000000000000000000034edd2a225f7f429a63e0f1d2084b9e0a93b53800000000000000000000000029d06ed7105c7552efd9f29f3e0d250e5df412cd`
+
+### Inputs
+**_implementation:** `0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD`
+
+**_proxy:** `0x034edD2A225f7f429A63E0f1D2084B9E0A93b538`
+
+## Tx #2: Upgrade `SystemConfig` proxy on Mode Sepolia
+Upgrades the `SystemConfig` proxy to the new implementation, featuring configurable EIP-1559 parameters.
+
+**Function Signature:** `upgrade(address,address)`
+
+**To:** `0xE7413127F29E050Df65ac3FC9335F85bB10091AE`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x99a88ec400000000000000000000000015cd4f6e0ce3b4832b33cb9c6f6fe6fc246754c200000000000000000000000029d06ed7105c7552efd9f29f3e0d250e5df412cd`
+
+### Inputs
+**_proxy:** `0x15cd4f6e0CE3B4832B33cB9c6f6Fe6fc246754c2`
+
+**_implementation:** `0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD`
+
+## Tx #3: Upgrade `SystemConfig` proxy on Metal Sepolia
+Upgrades the `SystemConfig` proxy to the new implementation, featuring configurable EIP-1559 parameters.
+
+**Function Signature:** `upgrade(address,address)`
+
+**To:** `0xF7Bc4b3a78C7Dd8bE9B69B3128EEB0D6776Ce18A`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x99a88ec40000000000000000000000005d63a8dc2737ce771aa4a6510d063b6ba2c4f6f200000000000000000000000029d06ed7105c7552efd9f29f3e0d250e5df412cd`
+
+### Inputs
+**_proxy:** `0x5D63A8Dc2737cE771aa4a6510D063b6Ba2c4f6F2`
+
+**_implementation:** `0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD`
+
+## Tx #4: Upgrade `SystemConfig` proxy on Zora Sepolia
+Upgrades the `SystemConfig` proxy to the new implementation, featuring configurable EIP-1559 parameters.
+
+**Function Signature:** `upgrade(address,address)`
+
+**To:** `0xE17071F4C216Eb189437fbDBCc16Bb79c4efD9c2`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x99a88ec4000000000000000000000000b54c7bfc223058773cf9b739cc5bd4095184fb0800000000000000000000000029d06ed7105c7552efd9f29f3e0d250e5df412cd`
+
+### Inputs
+**_proxy:** `0xB54c7BFC223058773CF9b739cC5bd4095184Fb08`
+
+**_implementation:** `0x29d06Ed7105c7552EFD9f29f3e0d250e5df412CD`
+
+## Tx #5: Upgrade `SystemConfig` proxy on Base Sepolia
```

Comment:
I think we need to treat Base Sepolia differently?

Reply:
Correct, they have a different owner on Sepolia so will need to be a different playbook